components/org.wso2.carbon.siddhi.error.handler.core/src/main/java/org/wso2/carbon/siddhi/error/handler/core/execution/RePlayer.java

```diff
@@ -78,27 +78,74 @@ private static void rePlay(ErrorEntry errorEntry) throws SiddhiErrorHandlerExcep
+                    case STORE_ON_TABLE_ADD:
+                        siddhiAppRuntime.getTableInputHandler(complexEventErrorEntry.getStreamName())
+                                .add(replayableTableRecord.getComplexEventChunk());
+                        break;
+                    case STORE_ON_TABLE_DELETE:
+                        siddhiAppRuntime.getTableInputHandler(complexEventErrorEntry.getStreamName())
+                                .delete(replayableTableRecord.getComplexEventChunk(),
+                                        replayableTableRecord.getCompiledCondition());
+                        break;
+                    case STORE_ON_TABLE_UPDATE:
+                        siddhiAppRuntime.getTableInputHandler(complexEventErrorEntry.getStreamName())
+                                .update(replayableTableRecord.getComplexEventChunk(),
+                                        replayableTableRecord.getCompiledCondition(),
+                                        replayableTableRecord.getCompiledUpdateSet());
+                        break;
+                    case STORE_ON_TABLE_UPDATE_OR_ADD:
+                        siddhiAppRuntime.getTableInputHandler(complexEventErrorEntry.getStreamName())
+                                .updateOrAdd(replayableTableRecord.getComplexEventChunk(),
+                                        replayableTableRecord.getCompiledCondition(),
+                                        replayableTableRecord.getCompiledUpdateSet(),
+                                        replayableTableRecord.getAddingStreamEventExtractor());
+                        break;
+                    default:
+                        throw new SiddhiErrorHandlerException("Unsupported ErrorOccurenceType of " +
+                            complexEventErrorEntry.getErrorOccurrence() + " to replay ComplexEventChunk in "
+                            + complexEventErrorEntry.getStreamName());
+                }
+            } else {
+                throw new SiddhiErrorHandlerException(
+                        "eventAsBytes present in the entry is invalid. It is expected to represent a " +
+                                ReplayableTableRecord.class);
+            }
+        } catch (IOException | ClassNotFoundException e) {
+            throw new SiddhiErrorHandlerException("Failed to get bytes as a ComplexEventChunk object.", e);
+        }
+    }
+
     private static void rePlayComplexEvent(ErrorEntry complexEventErrorEntry, SiddhiAppRuntime siddhiAppRuntime)
         throws SiddhiErrorHandlerException, InterruptedException {
         try {
             Object complexEvent = ErrorHandlerUtils.getAsObject(complexEventErrorEntry.getEventAsBytes());
             if (complexEvent instanceof ComplexEvent) {
-                InputHandler inputHandler = siddhiAppRuntime.getInputHandler(complexEventErrorEntry.getStreamName());
+                InputHandler inputHandler = siddhiAppRuntime.getInputHandler(complexEventErrorEntry
```

Comment:
Let's fix formatting to make this as one line

Reply:
Fixed in 379f207